.gitignore

```diff
@@ -28,3 +28,5 @@ debian/tmp
 debian/*.substvars
 debian/*.debhelper
 source/jormungandr/jormungandr/_version.py
+source/jormungandr/venv/*
```

Comment:
I personally think that it's better to put those in you own personal .gitignore file (since it does not apply for everybody, and in your case you'll want this for every venv directory